backend/app.js

```diff
@@ -74,7 +75,7 @@ app.use(function(req, res, next) {
   next();
 });
 
-var port = process.env.PORT || 3000;
+var port = process.env.PORT || 3005;
```

Comment:
Why was the port changed? I'm not sure how the setup startup script was made, but this could potentially break the production, so unless there is a very good reason for having 3005, it should remain 3000.